scripts/partition_manager/partition_manager.rst

```diff
@@ -772,19 +772,20 @@ This file is similar to the regular :file:`pm.yml` configuration files, except t
 
 You can set ``PM_STATIC_YML_FILE`` to contain exactly the static configuration you want to use.
 
-.. important::
-    |file_suffix_partition_manager_exception|
-
 If you do not set ``PM_STATIC_YML_FILE``, the build system will use the following order to look for files in your application source directory to use as a static configuration layout:
 
-* If a :term:`build type` is used, the following order applies:
+* If a :ref:`file suffix <app_build_file_suffixes>` is used, the following order applies:
 
-  1. If the file :file:`pm_static_<board>_<revision>_<buildtype>.yml` exists, it will be used.
-  #. Otherwise, if the file :file:`pm_static_<board>_<buildtype>.yml` exists, it will be used.
-  #. Otherwise, if the file :file:`pm_static_<buildtype>.yml` exists, it will be used.
+  1. If the file :file:`pm_static_<board>_<revision>_<suffix>.yml` exists, it will be used.
+  #. Otherwise, if the file :file:`pm_static_<board>_<suffix>.yml` exists, it will be used.
+  #. Otherwise, if the file :file:`pm_static_<suffix>.yml` exists, it will be used.
   #. Otherwise, if the file :file:`pm_static.yml` exists, it will be used.
```

Comment:
this has fallback, will need to manually check it works as expected but it should be this order:

- pm_static_<board>_<revision>_<suffix>.yml
- pm_static_<board>_<suffix>.yml
- pm_static_<suffix>.yml
- pm_static_<board>_<revision>.yml
- pm_static_<board>.yml
- pm_static.yml


Though now I suspect it might actually be:

- pm_static_<board>_<revision>_<suffix>.yml
- pm_static_<board>_<revision>.yml
- pm_static_<board>_<suffix>.yml
- pm_static_<board>.yml
- pm_static_<suffix>.yml
- pm_static.yml

Reply:
Have updated the list, also note that when hwmv2 is merged this will need updating to say board target